src/MeetingsSDKAdapter.js

```diff
@@ -667,88 +664,48 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
    * @param {string} ID ID of the meeting to mute audio
    */
   async handleLocalAudio(ID) {
-    const sdkMeeting = this.fetchMeeting(ID);
-
     try {
-      const isInSession = !!this.meetings[ID].remoteAudio;
-      const noAudio = !this.meetings[ID].disabledLocalAudio && !this.meetings[ID].localAudio.stream;
-      const audioEnabled = !!this.meetings[ID].localAudio.stream;
-      let state;
+      await this.updateMeeting(ID, async (meeting, sdkMeeting) => {
+        const isInSession = !!meeting.remoteAudio;
+        const audioEnabled = !!meeting.localAudio.stream;
+        let updates;
+
+        if (audioEnabled) {
```

Comment:
the `noAudio` case was handled before, it would be safer to keep it, as the audio can have 3 states:
- no audio available
- muted
- unmuted